content/docs/command-reference/add.md

```diff
@@ -153,10 +153,12 @@ not.
   > Additionally, this typically requires an external cache setup (see link
   > above).
 
-- `-o <path>`, `--out <path>` - destination `path` to make a local target copy,
-  or to [transfer](#example-transfer-to-cache) an external target into the cache
-  (and link to workspace). Note that this can be combined with `--to-remote` to
-  avoid storing the data locally, while still adding it to the project.
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to link
+  (or copy) a data target, which will now be tracked by DVC. Note that combining
```

Comment:
I'm not sure that this Note is relevant here. If `-o <path>` is generic enough to work in a regular case, we can remove the note.

Reply:
OK sure, having the note in the description and the examples should be enough. Simplified (and made consistent with `get/import -o`).